config/config.go

```diff
@@ -135,6 +140,11 @@ func (cfg *Configuration) validate(v *viper.Viper) []error {
 	if cfg.AccountDefaults.Events.Enabled {
 		glog.Warning(`account_defaults.events will currently not do anything as the feature is still under development. Please follow https://github.com/prebid/prebid-server/issues/1725 for more updates`)
 	}
+
+	if cfg.PriceFloors.Enabled {
+		glog.Warning(`cfg.PriceFloors.Enabled  will currently not do anything as price floors feature is still under development.`)
+	}
+
 	errs = cfg.Experiment.validate(errs)
 	errs = cfg.BidderInfos.validate(errs)
 	return errs
```

Comment:
Can you add validation for these new config fields in this PR?
```
func (pf *AccountPriceFloors) validate(errs []error) []error {

	if pf.EnforceFloorRate < 0 || pf.EnforceFloorRate > 100 {
		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.enforce_floors_rate should be between 0 and 100`))
	}
}
```

and then call this new function from this top-level `validate` function?

We can then claim that this PR covers all non-fetch logic defined in requirements section [4.1.1 Global and Account Config](https://docs.google.com/document/d/1YAt00RfQaU2Ugciqw1LnirLgjyUz9ibfyjfxWUHR_P0/edit#heading=h.vf5nn3il0n2h).

Reply:
updated